lib/geoengineer/gps/node.rb

```diff
@@ -172,40 +172,29 @@ def query_schema(node_type = "[a-zA-Z0-9\\-_\\/\\*]*")
     }
   end
 
+  def finder
+    @finder ||= GeoEngineer::GPS::Finder.new(all_nodes, {
+                                               project: @project,
+                                               environment: @environment,
+                                               configuration: @configuration,
+                                               node_type: @node_type,
```

Comment:
Do we want to initialize with `node_type` and `node_name` here? that would mean that `:::::` would equal itself

Reply:
yeah, it is a bit completionist, but you could look for all securitygroups